### PR TITLE
feat(curriculum): create performance quiz 

### LINKED ISSUE #56226: [HACKTOBERFEST 2024] - Performance Quiz
> [!IMPORTANT]
> This issue is part of our 2024 Hacktoberfest initiative. If you are interested in working on this issue, refer to our [top-level tasks](https://github.com/freeCodeCamp/freeCodeCamp/issues/56170) to see if this issue is claimed, or to claim it if it is not.

This issue is to create the Performance quiz.

The file you should be working on is found in `/curriculum/challenges/english/25-front-end-development/quiz-performance`.

**All quiz questions need to relate to one of these topics:**

- Real performance vs perceived performance
- Key performance concepts (source order, critical path, latency)
- How does a browser render a page
- Performance impacts on sustainability (energy efficiency, hardware and demand)
- "Time to useable"
- Improving percieved performance
- Key metrics (first contentful paint, cumulative layout shift)
- Performance Web APIs
- Improving CSS and JS performance

> [!CAUTION]
> Questions which do not fall under these topics will be rejected.

You are welcome to include code blocks in your questions, but please keep them short (four lines or less).

Please do NOT open a pull request until you have completed all questions. Your quiz should have **exactly 20 questions**.

### PR DESCRIPTION
**Pull request by AryanBhirud**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56226

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]

**Comment by lasjorg** [comment]
I know this was approved already, but could we include a question on INP as well?

The article even has a few quizzes at the bottom if we need inspiration.
https://web.dev/articles/inp

---

BTW, I'm out of the loop on these quizzes. What is the knowledge base they are founded upon?

Is it answers to questions we have given at some point in the curriculum, or just stand-alone quizzes with no direct connection to the curriculum?

**Comment by ilenia-magoni** [comment]
@lasjorg It's for the new curriculum, the lectures are WIP right now

**Comment by AryanBhirud** [comment]
@naomi-lgbt @ilenia-magoni Do you want me to add a question on Interaction to Next Paint (INP), or are these questions good enough?

**Comment by lasjorg** [comment]
@ilenia-magoni Then let me rephrase it, how does the camper know the answers?

Surely it isn't expected that challenges will be created that contain answers to all the questions asked? That would cause the quizzes to dictate the challenge content.

**Comment by ilenia-magoni** [comment]
@lasjorg the lectures should contain the knowledge needed for the quizzes

**Comment by lasjorg** [comment]
@ilenia-magoni OK, just seems a little backwards to me. But I get the motivation.

**Comment by jdwilkin4** [comment]
@AryanBhirud 

If you want to add one question on INP that is fine.

If you are already at 20 questions, then you will need to replace one of these questions with the INP one because each quiz should have a max of 20 questions.

**Comment by AryanBhirud** [comment]
@jdwilkin4 I have replaced a question with a new one on INP. Awaiting review.

**Review by lasjorg — APPROVED** [review]
